src/user-guide/running-workflows/workflow-run-modes.rst

```diff
@@ -0,0 +1,113 @@
+.. _workflow-run-modes:
+
+Workflow Run Modes
+------------------
+
+Cylc can run a workflow without running the real jobs, which may be
+useful when developing workflows. Tasks submitted
+using :ref:`workflow-run-modes.dummy` and
+:ref:`workflow-run-modes.simulation` can be used to understand how different outcomes
+and run lengths will affect the workflow.
+
+To start a workflow in one of these modes:
+
+.. code-block:: console
+
+   $ cylc play --mode=dummy <workflow-id>
+   $ cylc play --mode=simulation <workflow-id>
+
+.. admonition:: Limitations
+
+   * Workflow run mode is recorded in the workflow run database. Cylc will not let you
+     *restart* a dummy mode workflow in live mode, or vice versa. Instead,
+     install a new instance of the workflow and run it from scratch in the new mode.
+   * Workflows run in Dummy and Simulation mode do not allow the run mode of
+     individual tasks to be overridden by :ref:`task-run-modes.skip`
+
+
+.. _workflow-run-modes.dummy:
+
+Dummy Mode
+----------
+
+**Dummy mode** replaces real jobs with background jobs on the
+scheduler host which use ``sleep`` to simulate the run length according to
+the settings described for simulation mode.
+This avoids :term:`job runner` directives that request compute
+resources for real workflow tasks, and it allows any workflow configuration to
+be run locally in dummy mode.
+
+.. admonition:: Limitations
```

Comment:
(In this case I agree, these are limitations ... sensible, but limitations nonetheless)